api/v1beta2/integrationtestscenario_webhook.go

```diff
@@ -53,6 +53,28 @@ func (r *IntegrationTestScenario) ValidateCreate() (warnings admission.Warnings,
 					"param manually defined because it will be automatically generated"+
 					"by the integration service")
 		}
+		// we won't enable ITS if git resolver with url & repo+org
+		urlResolverExist := false
+		repoResolverExist := false
+		orgResolverExist := false
+
+		for _, gitResolverParam := range r.Spec.ResolverRef.Params {
+			if gitResolverParam.Name == "URL" && gitResolverParam.Value != "" {
+				urlResolverExist = true
+			}
+			if gitResolverParam.Name == "REPO" && gitResolverParam.Value != "" {
+				repoResolverExist = true
+			}
+			if gitResolverParam.Name == "ORG" && gitResolverParam.Value != "" {
+				repoResolverExist = true
+			}
+			if urlResolverExist && (repoResolverExist || orgResolverExist) {
```

Comment:
This should be probably outside of loop for better readability.

Reply:
taking this condition out of loop would work , but the way it's implemented here will apply fast exit on case condition is met then we return a proper error . @MartinBasti, isn't ?

Reply:
yes, but no if you address the case that org must be defined together with requirements to have both repo+org.

I'd say it's micro-optimization that is not worth

Reply:
right, 
I've fixed so the condition is moved out loop